recipes/motmetrics/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "motmetrics" %}
+{% set version = "1.2.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 7328d8468c948400b38fcc212f3e448bc1f2fdfc727e170d85a029e49f1cdbc6
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - enum34  # [py<3]
```

Comment:
Do you actually need this package on Python 2?

Reply:
(I'm working with Quinn) We don't for our specific application, no. But the package does support 2.7

Reply:
Conda-forge has by default stopped packaging for Python 2. So unless you have special need for it, it's enough to forego worrying about it.

Reply:
Yep! And it is far better for a package to be noarch

Reply:
So I think you can remove this line

Reply:
Hmm so looking at the requirements from the original package, it looks like the python version specification is also in the [requirements.txt](https://github.com/cheind/py-motmetrics/blob/develop/requirements.txt) (which makes sense, this was just grayskull output'd yaml) 

Should we still remove the version specification on it? More than happy to, but just figured the conda recipe should faithfully represent the requirements specified in the pip package

On the `noarch: python` bit -- I can add that in!

Reply:
You can write the version spec as `python >=3.6` e.g.